internal/cmd/beta/network-interface/create/create.go

```diff
@@ -0,0 +1,247 @@
+	Name             *string // <= 63 characters + regex  ^[A-Za-z0-9]+((-|_|\s|\.)[A-Za-z0-9]+)*$
+	NicSecurity      *bool
+	SecurityGroups   *[]string // = 36 characters + regex ^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates a network interface",
+		Long:  "Creates a network interface.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a network interface with allowed addresses "1.1.1.1,8.8.8.8,9.9.9.9" with ipv4 address "1.2.3.4" with label "key=value" with name "NAME" with security groups "UUID1,UUID2" with nic security enabled for network with ID "xxx"`,
+				`$ stackit beta network-interface create --network-id xxx --allowed-addresses "1.1.1.1,8.8.8.8,9.9.9.9" --ipv4 1.2.3.4 --labels key=value --name NAME --security-groups "UUID1,UUID2" --nic-security"`,
+			),
+			examples.NewExample(
+				`Create a network interface without allowed addresses with ipv4 address "1.2.3.4" with nic security disabled for network with ID "xxx"`,
+				`$ stackit beta network-interface create --network-id xxx --ipv4 1.2.3.4 --nic-security=false"`,
+			),
+			examples.NewExample(
+				`Create a network interface with allowed addresses "1.1.1.1,8.8.8.8" with ipv4 address "1.2.3.4" with ipv6 address "2001:0db8:85a3:08d3::0370:7344" with nic security enabled for network with ID "xxx"`,
+				`$ stackit beta network-interface create --network-id xxx --allowed-addresses "1.1.1.1,8.8.8.8" --ipv4 1.2.3.4 --ipv6 2001:0db8:85a3:08d3::0370:7344"`,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create a network interface for project %q?", model.ProjectId)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("create network area: %w", err)
+			}
+
+			return outputResult(p, model, resp)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), networkIdFlag, "Network ID")
+	cmd.Flags().StringSlice(allowedAddressesFlag, nil, "List of allowed IPs")
+	cmd.Flags().StringP(ipv4Flag, "i", "", "IPv4 address")
+	cmd.Flags().StringP(ipv6Flag, "s", "", "IPv6 address")
+	cmd.Flags().StringToString(labelFlag, nil, "Labels are key-value string pairs which can be attached to a network-interface. A label can be provided with the format key=value and the flag can be used multiple times to provide a list of labels")
```

Comment:
```suggestion
	cmd.Flags().StringToString(labelFlag, nil, "Labels are key-value pairs which can be attached to a network-interface. The expected value is a comma separated string (e.g 'key1=value1,key2=value2')" 
```

Reply:
Just a suggestion, but I think we should try to keep these flag descriptions as short as possible and providing an example is always a very good way of showing the expected format

Reply:
:+1:  I just copied this from the project command. 
There is a different description for the volume command and the same for the route command (but this is because I implemented the route command as well and copied the description from project command too).

I think we should use the same description everywhere. What do you think?

Reply:
I changed it to the `volume` version. Now we only need to change the `project` as well